modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -6,6 +6,7 @@ import pprint
 import re  # noqa: F401
 
 import six
+from decimal import Decimal
```

Comment:
Please remove this per the above comment

Reply:
Done

Reply:
Can you resolve this conversation?